frontend/assets/stylesheets/components/_social.scss

```diff
@@ -1,20 +1,58 @@
-.social {
-    height: rem(36px);
-    margin-left: rem(-10px);
+/* ==========================================================================
+   Social
+   ========================================================================== */
+
+.social-list {
+    @include clearfix();
+    margin: 0;
+    list-style: none;
 }
+    .social-list__item {
+        float: left;
+        margin-right: rem($gs-gutter / 4);
+    }
+    .social-list__item:last-child {
+        margin-right: 0;
+    }
+
+/* Social Actions
+   ========================================================================== */
+
+.social-action {
+    display: inline-block;
+    vertical-align: middle;
+    padding: 0; margin: 0;
+    width: 36px; height: 36px;
+    border-radius: 1000px;
+    background-color: $c-neutral2;
+    color: $white;
 
-.social__item {
-    float: left;
-    padding-left: rem(10px);
-    width: rem(36px + 10px);
+    &:hover {
+        background-color: darken($c-neutral2, 10%);
+    }
+
+    .icon-inline {
```

Comment:
I went down this route originally but it felt a little bit more explicit to have them separately as they server different functions. Going to move the vertical centring styles to a mixin I think.